options/options.js

```diff
@@ -0,0 +1,141 @@
+(() => {
+  const profilesWrapper = document.getElementById('profiles-wrapper');
+  const addButton = document.getElementById('add');
+  const PROFILES = 'profiles';
+
+  const uuid = () => {
+    return crypto.randomUUID();
+  }
```

Comment:
No need for this tiny wrapper IMO.

Reply:
Right, this used to be some `Math.random` until I read that crypto.randomUUID existed. I'll change that.